README.md

```diff
@@ -18,4 +18,7 @@ AC3: The output should be a string represents the multiplication table. Suppose
 1. Please draw the whole tasking diagram.
 2. Please declare all the functions according to your diagram.
 3. Please write a test case following given...when...then pattern for a leaf block of the diagram. Then implement the block to pass the test. Should only test the `render` method.
-4. Please repeat step 3 until all functions are implemented.
\ No newline at end of file
+4. Please repeat step 3 until all functions are implemented.
+
+# Task Diagram
+ ![img](https://s3.cn-north-1.amazonaws.com.cn/tws-upload/images/1666749322028-a277a425-93ba-4893-a397-c6da379b56bc.PNG)
```

Comment:
Put task diagram in README, it's professional.
But the name of variables in  task diagram is inappropriate, it shouldn't contain the tyle of variables, for example: startNumber.
And start can equal to end, the name of  "isStartLessThanEnd" has some problem.
Perhaps, the diagram of "getMultipicationTable" can be more detailed.